### ISSUE TITLE
Fix "Move" being the default option for manual import

### ISSUE THREAD

**Issue by dark-swordsman** [Sonarr/Sonarr#5005]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe

By default, sonarr uses local storage/cache to store the state of the "Move" or "Copy/Hardlink" dropdown. Shown here:
![image](https://user-images.githubusercontent.com/7909209/166097959-83668976-2325-4ec6-96ba-c5620d31bae2.png)

As a result, when using my laptop one time to manually hardlink some files, I ended up actually moving them instead. This left me wondering for a couple days why my movies (radarr, unrelated, but faces the same problem) and shows (sonarr) that I recently acquired had suddenly went missing on my original directory.

This is a problem, since I normally hardlink my files, and I also rename them. So in this instance, about 16 different videos have been renamed + moved. It is impossible for me to undo this action in Sonarr, so I am left with the choice of either: A) manually moving the files back and renaming them, or B) deleting them from my media directory and re-downloading them.

This is unacceptable behavior. Yes, you could argue that I should have been paying attention, _**but that is not how user interfaces work**_.

### Describe the solution you'd like

I think the best solution that _should_ be the lowest technical cost to implement, would be to simply have a modal pop up when the user goes to manually move files (if it is selected on move).

So for example:
- User has mode set to "Move"
- User clicks "Import" button 
- Modal pops up, warning user of a "move" operation
- User can click "cancel" or "import anyways" button

To add: Since we persist the local storage cookie for whether or not "Move" or "Hardlink/Copy" is selected, you could also have a cookie for for a "dissmiss message" on that modal. If a user wishes to hide the modal because they do Move operations anyways, they can choose to do that. Otherwise, the default state would be to just show the modal.

This should be a low cost feature since it's just adding another modal, cookie, and changing the import button to trigger a function that checks for the cookie, and either shows the modal or just moves anyways. 

### Describe alternatives you've considered

One alternative I considered was allowing the user to simply disable "Move" operations globally. This would be a toggle in the settings, and would prevent the user from ever doing a move operation. It could even remove the option from the manual import menu in the first place. Though, this is a much more involved feature, and was already denied in a previous issue.

### Anything else?

nope

**Comment by bakerboy448** [comment]
Related to / duplicate of comments in https://github.com/Sonarr/Sonarr/issues/4754

**Comment by dark-swordsman** [comment]
This isn't necessarily a duplicate. The idea of a warning prompt was never actually addressed in that thread. I think this is an important feature to prevent unfixable mistakes.

If there is a reason that it cannot be implemented beyond "No plans to...", I'd like to hear it. Otherwise, I fail to see how a feature like this shouldn't be implemented as a basic safeguard.

**Comment by bakerboy448** [comment]
Yup just linked them for reference.

It'd save quite a few headaches to swap the default - or have no default and have it required to be selected.

**Comment by markus101** [comment]
> I think the best solution that should be the lowest technical cost to implement, would be to simply have a modal pop up when the user goes to manually move files (if it is selected on move).

The lowest cost solution is changing the default, either to copy/hardlink or as baker mentioned, making the default nothing so you're forced to choose on a new browser.

> To add: Since we persist the local storage cookie for whether or not "Move" or "Hardlink/Copy" is selected, you could also have a cookie for for a "dissmiss message" on that modal. If a user wishes to hide the modal because they do Move operations anyways, they can choose to do that. Otherwise, the default state would be to just show the modal.

This just puts you back to the same place where you're at now. People will click import and then immediately click import anyways because they've been conditioned to do this in the past (you mentioned this in the last GHI as well). This is why years ago browsers moved away from easily being able to accept self signed and invalid certificates, requiring them them click/expand/take multiple actions.

A confirmation modal required when move is selected isn't something we're going to implement, same as we're not going to add a confirmation modal to copy/hardlink to warn users that importing may use 2x the disk space if copy/hardlink was the default.

When we previously discussed pretty much everyone felt that changing the default to copy/hardlink introduced a different issue in that by default you might use 2x the disk space (for a copy), when you're used to move being the default, but a default of nothing wasn't suggested, which seems like a reasonable compromise and prevents someone from doing the wrong thing if they fail to change the default.

TLDR: Change the default from `move` to a new `choose` option that requires the user to make a choice when run in a new browser and doesn't require a separate confirmation for every import.

**Comment by dark-swordsman** [comment]
Makes sense. I didn't consider having none as default.

Thanks for the consideration!
